### ISSUE TITLE
can filters work with expressions ?

### ISSUE THREAD

**Issue by booss** [Keats/tera#139]
Hi !
I'd like to be able to do this: {{ value / 60 | round }} but it seems impossible at the time.
I can do this {{ value | round }} but would it be possible to change the grammar and accept an expression instead of a ident only ?

I tried this for fun {% filter round %}{{ value / 60 }}{% endfilter %}
But it resulted in a panic

Thanks for your help !

**Comment by Keats** [comment]
That's a good idea! If anyone wants to do a PR for that, that should be relatively straightforward

**Comment by booss** [comment]
The PR addresses expressions with filters but I did not remove the identifier_with_filter part.
So something like {{ 1.5 + value | round }} would effectively do : 1.5 + round(value) which is not what is expected.

**Comment by Keats** [comment]
Hmm as you say, it is kind of ambiguous what should happen there, both `round(1.5 + value)` and `1.5 + round(value)` can be seen as the expected outcome. 
Not entirely sure what to do here, adding ambiguity is not a good idea imo.

Jinja2 does `1.5 + round(value)`:
```python

In [3]: t = Template("{{ 1.5 + val | round }}")

In [4]: t.render(val=1.1)
Out[4]: '2.5'

In [5]: t.render(val=1.7)
Out[5]: '3.5'
```
but you can use parenthesis to remove ambiguity
```
In [6]: t = Template("{{ (1.5 + val) | round }}")

In [7]: t.render(val=1.7)
Out[7]: '3.0'
```

After looking at that, I think a PR adding that would need to also add parenthesis to Tera but that becomes a much bigger beast. Pest is going to have a 1.0 soon-ish so I'd wait for that to land and Tera to move to that before starting something that big

**Comment by booss** [comment]
OK I agree, let's wait. Thanks

**Comment by Keats** [comment]
(reopening as I'm going to add that in 0.11)

**Comment by Keats** [comment]
Fixed as of 0.11.0-beta
